src/components/RemoveToDo.vue

```diff
@@ -1,15 +1,24 @@
 <template>
-  <button @click="removeTodo">x</button>
+  <button class="delete-button" @click="removeTodo">Delete</button>
 </template>
 
 <script setup lang="ts">
-
 const emit = defineEmits<{
-  (e: 'todoRemoved', type: any): void
-}>()
+  (e: "todoRemoved", type: any): void;
+}>();
 
 function removeTodo() {
   emit("todoRemoved", "");
```

Comment:
Why do you need to send an empty string? Can you just emit the todoRemoved event without any param? Please check the emit declaration too and remove the `type: any` part